### PR TITLE
Update to v0.5.9.1

### PR DESCRIPTION
**Pull request by FakeShemp**

### CODE REVIEW

File: net.lutris.Lutris.yml [hunk 1]
```diff
@@ -95,60 +95,60 @@ modules:
       - desktop-file-edit --set-key=X-Flatpak-RenamedFrom --set-value="lutris.desktop;" /app/share/applications/${FLATPAK_ID}.desktop
     sources:
       - type: archive
-        url: "https://github.com/lutris/lutris/archive/v0.5.8.4.tar.gz"
-        sha256: 32f07d321e582fa9775239ace59e63e9ec8438106472288a95db8de8212f1441
+        url: "https://github.com/lutris/lutris/archive/refs/tags/v0.5.9-beta1.tar.gz"
```
Comment (A6GibKm) [inline net.lutris.Lutris.yml:98]: Use https://github.com/lutris/lutris/archive/v0.5.9-beta1.tar.gz please.

**Review by A6GibKm — CHANGES_REQUESTED** [review]

**Comment by A6GibKm** [comment]
This could be split into two commits, one for the modules and one only updating lutris. If the first one builds we can merge it asap.

**Comment by FakeShemp** [comment]
bot, build

**Comment by FakeShemp** [comment]
bot, build

**Comment by A6GibKm** [comment]
I am getting an error 

    2021-10-12 15:25:11,376: No file /tmp/lutris-1db86942-b5ce-456b-9717-4fc39e57a8b0

**Comment by FakeShemp** [comment]
bot, build

**Comment by Unrud** [comment]
See #194

**Comment by A6GibKm** [comment]
Closed in favor of https://github.com/flathub/net.lutris.Lutris/pull/193. Thanks!

**Pull request closed without merging**